File: app/src/main/java/in/sudopk/vaishnavacalendar/VCalendar.java
package in.sudopk.vaishnavacalendar;

import java.util.List;

import in.sudopk.vaishnavacalendar.calendar.DayCalendar;

public class VCalendar {
    private List<DayCalendar> mCalendar;

    public VCalendar(final List<DayCalendar> calendar) {
        mCalendar = calendar;
    }

    public List<DayCalendar> getCalendar() {
        return mCalendar;
    }

    public boolean hasData() {
        return mCalendar != null && !mCalendar.isEmpty();
    }
}

File: app/src/main/java/in/sudopk/vaishnavacalendar/VCalendar.kt
package `in`.sudopk.vaishnavacalendar

import `in`.sudopk.vaishnavacalendar.calendar.DayCalendar
import com.google.gson.reflect.TypeToken
import java.lang.reflect.Type

typealias VCalendar = List<DayCalendar>

val VCalendarTypeToken : Type = object : TypeToken<VCalendar>() {}.type
